scripts/the-pied-pine-perf.yml

```diff
@@ -21,9 +21,7 @@ config:
     threshold: 100
   ensure:
       thresholds:
-        # I can't get these thresholds to work... they always fail no matter how high I set them
-        - http.response_time.p99: 100000
-        - http.response_time.p50: 100000
+        - browser.step.spinner_stage.median: 20000 # If actually implementing as part of a CI pipeline, then this asserts the spinner test did not go past 20 seconds.
```

Comment:
20000 probably isn't high enough, cold starts can take 30-40 seconds, but maybe revisit later if anyone wants to actually put this in a pipeline

Reply:
Did you try any of the p* thresholds?

Reply:
Since they are concerned with percentiles, we can give ourselves some more slack for the cold start, perhaps

Reply:
(and that's what wasn't working for me... always failed)

Reply:
no, but also, I don't know if there's a point right now, since we only have 2 users. the p* threshold will just be the median time no?

Reply:
if we implement p99 assertions, it would probably be within a particular http request, and not all requests. Like measuring the p99 of tile requests for example, since we know they'll have a large amount of requests during one session